python/spatialdata-codec-writer/docs/htj2k-wasm-encode-design.md

```
@@ -0,0 +1,110 @@
+| Contract | Detail |
+|----------|--------|
+| Zarr codec id (new writes) | `experimental.openjph_htj2k` |
+| Zarr codec id (legacy decode) | `experimental.imagecodecs_htj2k` |
+| Encoder label | `openjph-wasm` (manifest `encoder` field) |
+| Array metadata | Zarr v3; `codecs: [{ name, configuration: {} }]` |
+| Chunk bytes | Raw HTJ2K bitstream per 2D spatial plane (last two axes) |
+| Browser read | `registerExperimentalHtj2kCodec()` registers both ids |
+
+## Encode flow
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)